docs/PlatformSpecificInformation.md

```diff
@@ -55,6 +55,41 @@ var styles = StyleSheet.create({
 
 `Platform.OS` will be `ios` when running in iOS and `android` when running in an Android device or simulator.
 
+There is also a `Platform.select` method available, that given an object containing Platform.OS as keys,
+returns the value for the platform you are currently running on.
+
+```javascript
+var { Platform } = React;
```

Comment:
Just remove this line, we would assume that people know how to require Platform.

For a shipping in progress, it'll only be picked up if I re-shipit.

We can just leave it for now, or kill it in another commit once it lands